bitrise.yml

```diff
@@ -36,26 +38,28 @@ workflows:
         - force_provisioning_profile: "$BITRISE_PROFILE_ID"
         - verbose_log: 'no'
         - configuration: Debug
-    - xcode-test@2:
+    - xcode-test@4:
         inputs:
         - generate_code_coverage_files: 'yes'
         - should_build_before_test: 'no'
-        - should_retry_test_on_fail: 'yes'
+        - test_repetition_mode: up_until_maximum_repetitions
```

Comment:
oh I didn't know this 👍

Reply:
Yes, the old parameter was was deprecated.